configs/sst_desktop-sysprof.yaml

```diff
@@ -0,0 +1,12 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Sysprof
+  description: System-wide profiling tool
+  maintainer: sst_desktop
+
+  packages:
+  - sysprof
+
+  labels:
+  - eln
```

Comment:
Shouldn't this also be marked for `c10s` too?

Reply:
c10s is still eln, no? I not, then I'm not aware as there wasn't any RH internal or Fedora announcement.

Reply:
It should be, but then I started seeing `c10s` labels show up on some of the yamls. I don't know what's going on...

Reply:
Some people are just jumping the gun, that's all.  c10s is not yet split from eln, and when that split does occur, any configs with eln will get a c10s label added.  No harm in adding it now, but it's not required yet.